shared/src/main/scala/weaponregex/model/regextree/Node.scala

```diff
@@ -57,9 +61,15 @@ case class Lookaround(expr: RegexTree, isPositive: Boolean, isLookahead: Boolean
       ")"
     )
 
+// Comments like these are also nice as a ScalaDoc so I know what I'm auto-completing :)
```

Comment:
Will surely be properly documented as ScalaDoc in the documenting phase at the end of the project. This is also the case for other quick comments we made across the code.